Resources/Private/Language/locallang.xlf

```diff
@@ -263,6 +289,14 @@
 		<source>File(s)</source>
 	  </trans-unit>
 
+	  <trans-unit id="tx_ats.upload.back">
+		<source>Back</source>
+	  </trans-unit>
+
+	  <trans-unit id="tx_ats.summary.back">
+		<source>Back</source>
```

Comment:
I guess these two can be combined.

Reply:
Not necessarily, as someone could want to modify them to "back to form" etc. Then the labels would be different.